package.json

```diff
@@ -61,16 +61,17 @@
     "husky": "^4.2.5",
     "jasmine-core": "~3.6.0",
     "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~5.0.0",
+    "karma": "~6.3.3",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage-istanbul-reporter": "~3.0.2",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
-    "ng-packagr": "^11.0.0",
+    "ng-packagr": "^12.0.3",
     "protractor": "~7.0.0",
     "ts-node": "~8.3.0",
     "tslint": "~6.1.0",
-    "typescript": "~4.1.5"
+    "typescript": "~4.1.5",
+    "ws": "^7.4.6"
```

Comment:
Question: Why do we add new dependency?

Reply:
Hi, thanks for the question! It is because there was a moderate vulnerability associated with that dependency, it was fixed by adding it into the list. 
file:///C:/Users/bma/Desktop/repos/reports/package/ngx-scroll-to/vulnerability_initial_ngx-scroll-to.html